iothub_client/inc/iothub_client_version.h

```diff
@@ -9,7 +9,7 @@
 #define IOTHUB_CLIENT_VERSION_H
 
 /** @brief Macro representing current IoT Hub Client SDK version. */
-#define IOTHUB_SDK_VERSION "1.11.0"
+#define IOTHUB_SDK_VERSION "1.12.0"
```

Comment:
We use semver: I assume we increased to 1.12 because of the new public APIs in blob upload? /cc @ewertons